instrumentation/opentelemetry-instrumentation-aws-lambda/src/opentelemetry/instrumentation/aws_lambda/__init__.py

```diff
@@ -377,11 +393,12 @@ def _instrument(self, **kwargs):
         ) = lambda_handler.rsplit(".", 1)
 
         flush_timeout_env = os.environ.get(
-            OTEL_INSTRUMENTATION_AWS_LAMBDA_FLUSH_TIMEOUT, ""
+            OTEL_INSTRUMENTATION_AWS_LAMBDA_FLUSH_TIMEOUT, None
         )
         flush_timeout = 30000
         try:
-            flush_timeout = int(flush_timeout_env)
+            if flush_timeout_env is not None:
```

Comment:
Why not `if flush_timeout_env`? Maybe you want to guard against the `""` environment variable. But perhaps users would prefer a warning log if they set this variable (which should be an `int`) to an empty string?